thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomalydetection/performanceEvaluation/IntervalUtils.java

```diff
@@ -0,0 +1,78 @@
+package com.linkedin.thirdeye.anomalydetection.performanceEvaluation;
+
+import com.linkedin.thirdeye.api.DimensionMap;
+import com.linkedin.thirdeye.datalayer.dto.MergedAnomalyResultDTO;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Stack;
+import org.joda.time.Interval;
+
+
+public class IntervalUtils {
```

Comment:
Does joda.time.Interval support merge functionality? I saw the same logic has been implement couple of places within 3rd eye repository: for example in Merge Anomalies. We should think about unify (merge intervals) logic.